ui/src/main/js/field/input/SelectInput.js

```diff
@@ -1,24 +1,47 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import Select from 'react-select';
-import LabeledField from 'field/LabeledField';
+import LabeledField, { LabelFieldPropertyDefaults } from 'field/LabeledField';
 
 function noOptionsMessage() {
     return null;
 }
 
-class SelectInput extends Component {
-    render() {
-        const {
-            onChange, id, inputClass, options, isSearchable, placeholder, value, removeSelected, hasMultipleValues, components, selectSpacingClass, readOnly, clearable
-        } = this.props;
-        const selectClasses = `${selectSpacingClass} d-inline-flex p-2`;
-        let closeOnSelect = true;
-        if (hasMultipleValues) {
-            closeOnSelect = false;
-        }
+const SelectInput = ({
```

Comment:
Is this still used? I know a lot of our code changed to using the dynamic select component.

Reply:
No I will remove it.